api/messages.py

```diff
@@ -41,188 +42,131 @@ def send_message(self, user, text):
                 return
             except:
                 pass
-
+    
     def welcome(self, user):
+        ''' User enabled Telegram Notifications'''
         lang = user.profile.telegram_lang_code
 
-        # In weird cases the order cannot be found (e.g. it is cancelled)
-        queryset = Order.objects.filter(maker=user)
-        order = queryset.last()
-
-        print(str(order.id))
         if lang == 'es':
-            text = f'Hola {user.username}, te enviaré un mensaje cuando tu orden con ID {str(order.id)} haya sido tomada.'
+            text = f'Hola {user.username}, te enviaré notificaciones sobre tus órdenes en RoboSats.'
         else:
-            text = f"Hey {user.username}, I will send you a message when someone takes your order with ID {str(order.id)}."
+            text = f"Hey {user.username}, I will send you notifications about your RoboSats orders."
```

Comment:
Maybe not for this PR but probably it's worth it to include also i18n on the backend https://github.com/danhper/python-i18n

Reply:
Yes, agree. There is no internationalization implemented on the backend yet, but would be very useful for the Telegram messages.